.github/workflows/csv-coverage-update.yml

```
@@ -5,6 +5,10 @@ on:
   schedule:
     - cron: "0 0 * * *"
 
+permissions:
+  contents: read
+  security-events: write
```

Is this right? Should it be this?

```suggestion
  pull-requests: write
```

`scripts/library-coverage/create-pr.py` will open a new PR. I don't see anywhere that it needs to upload security-related things. Might be mistaken, though.

@jsoref can you run this in your fork to see if it works?

---

Yes: https://github.com/check-spelling-sandbox/codeql/actions/runs/7824516174/job/21347184874

---

Actually, I'm not entirely certain what is necessary to make it generate a PR (maybe I need to borrow an older file?), but conceptually, that change is correct.